[2025-02-14T21:37:35.665+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-14T21:37:35.690+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_run_operator_dag.task2 manual__2025-02-14T21:37:27.090590+00:00 [queued]>
[2025-02-14T21:37:35.700+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_run_operator_dag.task2 manual__2025-02-14T21:37:27.090590+00:00 [queued]>
[2025-02-14T21:37:35.701+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-14T21:37:35.713+0000] {taskinstance.py:2890} INFO - Executing <Task(TriggerDagRunOperator): task2> on 2025-02-14 21:37:27.090590+00:00
[2025-02-14T21:37:35.719+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=828) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-14T21:37:35.721+0000] {standard_task_runner.py:72} INFO - Started process 829 to run task
[2025-02-14T21:37:35.722+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_run_operator_dag', 'task2', 'manual__2025-02-14T21:37:27.090590+00:00', '--job-id', '850', '--raw', '--subdir', 'DAGS_FOLDER/11_run_operator_dag.py', '--cfg-path', '/tmp/tmpceydsmrq']
[2025-02-14T21:37:35.723+0000] {standard_task_runner.py:105} INFO - Job 850: Subtask task2
[2025-02-14T21:37:35.766+0000] {task_command.py:467} INFO - Running <TaskInstance: call_run_operator_dag.task2 manual__2025-02-14T21:37:27.090590+00:00 [running]> on host ab74d4d90d32
[2025-02-14T21:37:35.851+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_run_operator_dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T21:37:27.090590+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T21:37:27.090590+00:00'
[2025-02-14T21:37:35.852+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-14T21:37:35.947+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-14T21:37:35.947+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=call_run_operator_dag, task_id=task2, run_id=manual__2025-02-14T21:37:27.090590+00:00, execution_date=20250214T213727, start_date=20250214T213735, end_date=20250214T213735
[2025-02-14T21:37:36.017+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-14T21:37:36.045+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-14T21:37:36.047+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
